public/static/docs/api-reference/get_url.md

```diff
@@ -0,0 +1,85 @@
+# dvc.api.get_url()
+
+Return the URL to the storage location of the target data (`repo`/`path`).
+
+## Signature
+
+```py
+get_url(path, repo=None, rev=None, remote=None)
+```
+
+## Description
+
+Returns the full URL string (`str` type) to the physical location (in a
+[DVC remote](/doc/command-reference/remote)) where a target <abbr>data
+artifact</abbr> specified by its `path` in a `repo` (<abbr>DVC project</abbr>)
+is stored.
+
+⚠️ Note that the returned URL is formed by analyzing the corresponding
+[DVC-file](/doc/user-guide/dvc-file-format) (see [Examples](#examples) below).
+**There is no guarantee that the file actually exists in that location**. Please
```

Comment:
instead we should write something like - `this call does not check for the existence of the files or directory on the remote storage`.

Reply:
Good call. I moved the explanation about how the URL is formed to the 1st paragraph and simplified this note. See f13b3111